aleph-client/src/pallets/multisig.rs

```diff
@@ -243,6 +247,38 @@ impl ContextState for Ongoing {}
 pub enum Closed {}
 impl ContextState for Closed {}
 
+// Wrapper for AccountId that implements Hash.
```

Comment:
Previously we had `pub fn approvers(&self) -> &HashSet<AccountId>`, but now AccountId is not `Hash`, so I've chosen to modify the code this way.

Reply:
Rolled back as we're using substrate's AccountId instead